llama_stack/providers/inline/post_training/huggingface_ilab/config.py

```diff
@@ -8,6 +8,9 @@
 
 from pydantic import BaseModel
 
+MINIMUM_VRAM_REQUIREMENT = 9.6e10 # 96 GB?
+GPU_MAXIMUM_POWER_DRAW = 72 # 72w maximum power draw?
+GPU_MAXIMUM_UTILIZATION = 20 # if 20% of GPU is already bring used percentage wise, error.
```

Comment:
I think util should be really small, otherwise we can consider the card occupied